### PR TITLE
Change lexer regex to be simpler

### PR DESCRIPTION
**Pull request by ydah**

### CODE REVIEW

File: lib/lrama/lexer.rb [hunk 1]
```diff
@@ -71,8 +71,7 @@ def lex_token
           # noop
         when @scanner.scan(/\/\*/)
           lex_comment
-        when @scanner.scan(/\/\//)
-          @scanner.scan_until(/\n/)
+        when @scanner.scan(/\/\/.*?\n/)
```
Comment (yui-knk) [inline lib/lrama/lexer.rb:74]: This is changed by https://github.com/ruby/lrama/pull/245. Could you rebase (or merge) master branch and use current master implementation?

Comment (ydah) [reply lib/lrama/lexer.rb]: Sorry for the delay. I updated this PR.

**Comment by yui-knk** [comment]
Thank you!

File: lib/lrama/lexer.rb (hunk 1 above)
Comment (yui-knk) [reply lib/lrama/lexer.rb]: No worries!

**Pull request merged**